test/e2e/framework/pod/delete.go

```diff
@@ -39,6 +39,12 @@ const (
 func DeletePodOrFail(c clientset.Interface, ns, name string) {
```

Comment:
Please document the change of semantics, for example as in `DeletePodWithWait` by adding "Resilient to the pod
not existing.".

Reply:
done